www/templates/series-diff.html

```diff
@@ -4,6 +4,22 @@
 <link rel="stylesheet" href="/static/jsdifflib/diffview.css">
 <script src="/static/jsdifflib/difflib.js"></script>
 <script src="/static/jsdifflib/diffview.js"></script>
+<style>
+.diff { width: 100%; table-layout: fixed; margin-top: 1em; border: 1px solid #999; }
+.diff th, .diff td { width: 50%; border: 1px solid #999; vertical-align: top; }
+
+.diff th { display: none; }
+.diff th.author { display: table-cell; }
+.diff th.texttitle { display: table-cell; }
+
+.diff thead th, .diff thead td { background: #ddd; }
+.diff tbody td { border-bottom-color: #ddd; }
+.diff td.delete { border-bottom-color: #ebb; background: #fcc; }
+.diff td.insert { border-bottom-color: #beb; background: #cfc; }
+.diff td.equal { border-bottom-color: #eee; background: #fff; }
+.diff td.empty { border-bottom-color: #ddd; background: #eee; }
+.diff div { overflow-x: hidden; white-space: pre; }
+</style>
```

Comment:
Let's have a `series-diff.css` for these styles? And can we make a separate commit so that the `jsdifflib` customizations have its own commit which will make it easier if we rebase it later.